DIFFERENCES.md

```diff
@@ -17,3 +17,9 @@
   Go version will handle this as "API error" while ruby version will fail to
   parse the response and handle this as "JSON error".
 - The Go HTTP client tries to reuse connections with keep-alive.
+- Docs for GET APIs call for URL-encoded query params. Ruby version sends a
```

Comment:
Is this related to rollback somehow?

Reply:
I noticed this while comparing debug traces of old and new version. IIRC this was visible in `showProduct()` GET call.

Reply:
ok, yes I checked and showProduct() is the only api call that uses toQuery(). I think we should specify the difference is between Go showProduct() and Ruby show_product(), because as it stands the reader would have to go compare the API doc, Go code and Ruby code, to find which call(s) this is about

Reply:
added note about `showProduct()`